bip-musig2.mediawiki

```diff
@@ -249,6 +249,17 @@ The following conventions are used, with constants as defined for [https://www.s
 
 === Key Generation ===
 
+==== Key Generation of participating Signers ====
+
+<div>
+Algorithm ''PlainPubKey(sk)'':
```

Comment:
```suggestion
Algorithm ''IndividualPubKey(sk)'':
```

Can we call it like this? I think "plain" is somewhat confusing, we use it to distinguish from "x-only", but that distinction is orthogonal to what we add here. For example,  we also have "GetPlainPubKey()", which refers to the *aggregate* pubkey in plain format.

---

We could add a footnote that this matches PubKey from BIP340.

Reply:
Do you mean add a footnote that this does _not_ match BIP340's PubKey?

Reply:
Oh, I actually meant that it matches. But yeah, sorry, brain fart. :)

Maybe we could footnote instead that it matches the plain key generation traditionally used for ECDSA in Bitcoin (if you agree with removing the "PlainPubKey" name.)

Reply:
Ah that's a good idea, done. Also replaced  `plain_pk_gen` with `individual_pk` in the python code.